renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableRateLimiting",
+    ":maintainLockFilesWeekly",
+    ":semanticCommits"
+  ],
+  "postUpdateOptions": ["pnpmDedupe"],
+  "rebaseWhen": "conflicted"
```

Comment:
As long as the PR is mergeable, updating the PR over and over will just exhaust the CI.